docs/examples/physics_using_pybullet/viz_domino.py

```diff
@@ -0,0 +1,226 @@
+"""
+=========================
+Domino Physics Simulation
+=========================
+
+This example simulation shows how to use pybullet to render physics simulations
+in fury. In this example we specifically render a series of Dominoes which are
+under Domino Effect.
+
+First some imports.
+"""
+import numpy as np
+from fury import window, actor, ui, utils
+import itertools
+import pybullet as p
+
+# Next, we initialize a pybullet client to render the physics.
+# We use `DIRECT` mode to initialize pybullet without a GUI.
+client = p.connect(p.DIRECT)
+
+# Apply gravity to the scene.
+p.setGravity(0, 0, -10, physicsClientId=client)
+
+###############################################################################
+# Base Plane Parameters
+base_size = np.array([15, 15, 0.2])
+base_color = np.array([1, 1, 1])
+base_position = np.array([0, 0, -0.1])
+base_orientation = np.array([0, 0, 0, 1])
+
+# Render a BASE plane to support the Dominoes.
+base_actor = actor.box(centers=np.array([[0, 0, 0]]),
+                       directions=[0, 0, 0],
+                       scales=base_size,
+                       colors=base_color)
+
+# half of the actual size.
+base_coll = p.createCollisionShape(p.GEOM_BOX,
+                                   halfExtents=base_size / 2)
+
+base = p.createMultiBody(
+    baseCollisionShapeIndex=base_coll,
+    basePosition=base_position,
+    baseOrientation=base_orientation)
+
+p.changeDynamics(base, -1, lateralFriction=1, restitution=0.5)
+
+###############################################################################
+# We define some global parameters of the Dominoes so that its easier for
+# us to tweak the simulation.
+
+domino_mass = 0.5
+domino_size = np.array([0.1, 1, 2])
+
+number_of_dominoes = 10
```

Comment:
If I change the number of dominoes,  it will be good if you could update the size of the base. Otherwise, it looks strange

Reply:
you can check the simulation when you change the value to 20.